mcp/pacemaker.sysconfig

```diff
@@ -1,95 +1,102 @@
+# Specify an alternate syslog target for NOTICE (and higher) messages.
+# Use 'none' to disable - not recommended.
+# The default value is 'daemon'.
 # PCMK_logfacility=none|daemon|user|local0|local1|local2|local3|local4|local5|local6|local7
 
-# Send all messages up-to-and-including the configured priority to syslog
-# A value of 'info' will be far too verbose for most installations and 'debug' is almost certain to send you blind
-# The default value is 'notice'
+# Send all messages up-to-and-including the configured priority to syslog.
+# A value of 'info' will be far too verbose for most installations and 'debug'
+# is almost certain to send you blind.
+# The default value is 'notice'.
 # PCMK_logpriority=emerg|alert|crit|error|warning|notice|info|debug
 
-# Log all messages from a comma-separated list of functions
+# Log all messages from a comma-separated list of functions.
 # PCMK_trace_functions=function1,function2,function3
 
-# Log all messages from a comma-separated list of files (no path)
-# Supports wildcards eg. PCMK_trace_files=prefix*.c
+# Log all messages from a comma-separated list of files (no path).
+# Supports wildcards eg. PCMK_trace_files=prefix*.c.
 # PCMK_trace_files=file.c,other.h
 
-# Log all messages matching comma-separated list of formats
+# Log all messages matching comma-separated list of formats.
 # PCMK_trace_formats="Sent delete %d"
 
-# Log all messages from a comma-separated list of tags
+# Log all messages from a comma-separated list of tags.
 # PCMK_trace_tags=tag1,tag2
 
 # Dump the blackbox whenever the message at function and line is printed
-# eg. PCMK_trace_blackbox=te_graph_trigger:223,unpack_clone:81
+# eg. PCMK_trace_blackbox=te_graph_trigger:223,unpack_clone:81.
 # PCMK_trace_blackbox=fn:line,fn2:line2,...
 
-# Enable blackbox logging globally or per-subsystem
-# The blackbox contains a rolling buffer of all logs (including info+debug+trace)
-# and is written after a crash, assertion failure and/or when SIGTRAP is received
+# Enable blackbox logging globally or per-subsystem.
+# The blackbox contains a rolling buffer of all logs (incl. info+debug+trace)
+# and is written after a crash, assertion failure and/or when SIGTRAP is
+# received.
 #
-# The blackbox recorder can also be enabled for Pacemaker daemons at runtime by 
-# sending SIGUSR1 (or SIGTRAP), and disabled by sending SIGUSR2
+# The blackbox recorder can also be enabled for Pacemaker daemons at runtime
+# by sending SIGUSR1 (or SIGTRAP), and disabled by sending SIGUSR2.
 #
 # Multiple subsystems may me listed separated by commas
-# eg. PCMK_blackbox=crmd,pengine
+# eg. PCMK_blackbox=crmd,pengine.
 # PCMK_blackbox=yes|no|crmd|pengine|cib|stonith-ng|attrd|pacemakerd
 
 #==#==# Advanced use only
 
 # Enable this for compatibility with older corosync (prior to 2.0)
-# based clusters which used the nodes uname as its uuid also
+# based clusters which used the nodes uname as its uuid also.
 # PCMK_uname_is_uuid=no
```

Comment:
While yes/no are accepted, true/false would be more consistent with the code's usage. Maybe: 'If the cluster uses an older version of corosync (prior to 2.0), set this to "true", which will use a node's uname as its UUID. The default, "false", is appropriate for newer versions of corosync, and will use a node's corosync ID as its UUID. It is ignored by clusters that do not use corosync.'